changelogs/fragments/netcommon_ref_update.yaml

```
@@ -0,0 +1,4 @@
+---
+major_changes:
+  - "Minimum required ansible.netcommon version is 2.5.0."
```

```suggestion
  - "Minimum required ansible.netcommon version is 2.5.1."
```
I don't see this enforced?